compiler/rustc_hir/src/lang_items.rs

```diff
@@ -67,15 +74,16 @@ macro_rules! language_item_table {
             }
         }
 
+        /// All of the language items in the current crate, defined or not.
```

Comment:
Should probably clarify this to say either "all lang items defined in the current crate" or "all lang items available in the current crate". I don't know which one is correct though :)

cc https://github.com/rust-lang/rust/pull/82641#r584351324

Reply:
It includes items from other crates: https://github.com/rust-lang/rust/blob/861872bc453bde79b83ff99d443d035225f10e87/compiler/rustc_passes/src/lang_items.rs#L198-L203

Reply:
Improved in d31f70c87c4.